Configurations/20-windows-posix.conf

```diff
@@ -0,0 +1,111 @@
+## -*- mode: perl; -*-
+# Copyright 2015-2018 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the OpenSSL license (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+(
+#### MinGW
+    "mingw" => {
```

Comment:
mingw is *not* "windows posix" thing, it straight windows, just non-ms compiler.

Reply:
With the mingw shell, it does provide a POSIXly environment, no?  But feel free to suggest a better file name...

Reply:
There is no "mingw shell". In most cases it's MSYS2 shell with mingw cross-compiler on $PATH. [In the context note that MSYS2 is windows posix" thing, and it *is* different from mingw.] As for cases other than most, it can as well be Linux shell with mingw cross-compiler on $PATH. That's how it's done on travis.

As for name suggestion, how about 20-windows,conf with ms and mingw targets.

Reply:
So you're suggesting on `20-windows.conf` containing all the (non-CE) VC- config targets and all the mingw config targets.  Do I get that correctly?  What do you want done with cygwin?